### ISSUE TITLE
Insert Size

### ISSUE THREAD

**Issue by alexjironkin** [stephenturner/covcalc#1]
Nice looking app,

But I think you are missing an important parameter: insert size. If your insert size is greater than read length, then your app works just fine. But very often insert sizes are less than that, which means reads overlap, and you get less useful information.

As a result you would need more reads to meaningfully cover your genome, instead of covering a base twice from the same DNA fragment sequenced forward and reverse.

**Comment by stephenturner** [comment]
Good call, it was on my to-do list. What's the more conventional way to specify - fragment size, or insert size (between the seq reads)? I feel like fragment size may be more intuitive?

**Comment by alexjironkin** [comment]
I think insert size is used more; BAMs and SAMs have it. But I agree, fragment size is more intuitive.
